src/SFA.DAS.AdminService.Web.Tests/Controllers/Dashboard/DashboardControllerTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AdminService.Settings;
+using SFA.DAS.AdminService.Web.Controllers;
+using SFA.DAS.AdminService.Web.Controllers.Roatp.Apply;
+using SFA.DAS.AdminService.Web.Infrastructure;
+using SFA.DAS.AdminService.Web.Services.Gateway;
+using SFA.DAS.AdminService.Web.Validators.Roatp;
+using SFA.DAS.AdminService.Web.ViewModels;
+using SFA.DAS.AssessorService.ApplyTypes;
+
+namespace SFA.DAS.AdminService.Web.Tests.Controllers.Dashboard
+{
+    [TestFixture]
+    public class DashboardControllerTests
+    {
+        private DashboardController _controller;
+        private Mock<IApplicationApiClient> _apiClient;
+        private Mock<IWebConfiguration> _configuration;
+
+        private ApplicationReviewStatusCounts _statusCounts;
+        private string _dashboardUrl;
+
+
+        [SetUp]
+        public void Setup()
+        {
+            _apiClient = new Mock<IApplicationApiClient>();
+            _configuration = new Mock<IWebConfiguration>();
+            _statusCounts = new ApplicationReviewStatusCounts
+            {
+                OrganisationApplicationsNew = 1,
+                OrganisationApplicationsInProgress = 2,
+                OrganisationApplicationsHasFeedback = 3,
+                OrganisationApplicationsApproved = 4,
+                StandardApplicationsNew = 5,
+                StandardApplicationsInProgress = 6,
+                StandardApplicationsHasFeedback = 7,
+                StandardApplicationsApproved = 8
+            };
+            _dashboardUrl = "http://dashboard";
+
+            _controller = new DashboardController(_apiClient.Object,_configuration.Object);
+        }
+
+        [Test]
+        public async Task confirm_viewModel_is_as_expected()
+        {
+            _apiClient.Setup(x => x.GetApplicationReviewStatusCounts()).ReturnsAsync(_statusCounts);
+            _configuration.Setup(c => c.RoatpOversightBaseUrl).Returns(_dashboardUrl);
```

Comment:
Could this line be moved into setup? Doesn't look like it does anything functional for the test